index.js

```diff
@@ -608,6 +794,24 @@
     'ninth'
   ];
 
+  //  ordinal :: Integer -> String
+  var ordinal = function(idx) {
+    return ordinals[idx];
+  };
+
+  //  ordinalList :: [Integer] -> String
+  var ordinalList = function(indexes) {
+    if (indexes.length === 1) {
+      return ordinals[indexes[0]];
+    } else {
+      var init = indexes.slice(0, -1);
+      return (
+        map(init, ordinal).join(', ') + ' and ' +
+        ordinals[indexes[indexes.length - 1]]
+      );
+    }
+  };
+
```

Comment:
I'm not sure we'll end up needing this function with the new error messages, but I've updated it in the meantime